### PR TITLE
add interactive examples for intl/segmenter et al.

### PR DESCRIPTION
**Pull request by SphinxKnight**
#### Summary

Adding interactive examples for `Intl.Segmenter` and related pages

#### Motivation

https://github.com/mdn/interactive-examples/pull/1987 has been merged

#### Supporting details

https://github.com/mdn/content/pull/8402/

#### Metadata
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/intl/segmenter/index.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segmenter
 
 The **`Intl.Segmenter`** object enables locale-sensitive text segmentation, enabling you to get meaningful items (graphemes, words or sentences) from a string.
 
+{{EmbedInteractiveExample("pages/js/intl-segmenter.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segmenter/index.md:16]: Hi! @SphinxKnight.

Any reason for the trailing space?

```suggestion
{{EmbedInteractiveExample("pages/js/intl-segmenter.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segmenter/resolvedoptions/index.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segmenter.resolvedOptions
 
 The **`Intl.Segmenter.prototype.resolvedOptions()`** method returns a new object with properties reflecting the locale and granularity options computed during the initialization of this [`Intl.Segmenter`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter) object.
 
+{{EmbedInteractiveExample("pages/js/intl-segmenter-prototype-resolvedoptions.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segmenter/resolvedoptions/index.md:16]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segmenter-prototype-resolvedoptions.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segmenter/segment/index.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segmenter.segment
 
 The **`Intl.Segmenter.prototype.segment()`** method segments a string according to the locale and granularity of this [`Intl.Segmenter`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter) object.
 
+{{EmbedInteractiveExample("pages/js/intl-segmenter-prototype-segment.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segmenter/segment/index.md:16]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segmenter-prototype-segment.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segmenter/segmenter/index.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ browser-compat: javascript.builtins.Intl.Segmenter.constructor
 
 The **`Intl.Segmenter()`** constructor creates [`Intl.Segmenter`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter) objects that enable locale-sensitive text segmentation.
 
+{{EmbedInteractiveExample("pages/js/intl-segmenter.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segmenter/segmenter/index.md:17]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segmenter.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segmenter/supportedlocalesof/index.md [hunk 5]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segmenter.supportedLocalesOf
 
 The **`Intl.Segmenter.supportedLocalesOf()`** method returns an array containing those of the provided locales that are supported without having to fall back to the runtime's default locale.
 
+{{EmbedInteractiveExample("pages/js/intl-segmenter-supportedlocalesof.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segmenter/supportedlocalesof/index.md:16]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segmenter-supportedlocalesof.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segments/@@iterator/index.md [hunk 6]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segments.@@iterator
 
 The **`Intl.Segments.@@iterator`** method is part of [the iterable protocol](/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#the_iterable_protocol), which returns a new iterator object that can iterate over the entries in an `Intl.Segmenter` object.  Each entry is returned as an object.
 
+{{EmbedInteractiveExample("pages/js/intl-segments-prototype-@@iterator.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segments/@@iterator/index.md:16]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segments-prototype-@@iterator.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segments/containing/index.md [hunk 7]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segments.containing
 
 The **`Intl.Segments.containing()`** method returns an object describing the segment in the string that includes the code unit at the specified index.
 
+{{EmbedInteractiveExample("pages/js/intl-segments-prototype-containing.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segments/containing/index.md:16]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segments-prototype-containing.html")}}
```

File: files/en-us/web/javascript/reference/global_objects/intl/segments/index.md [hunk 8]
```diff
@@ -13,6 +13,8 @@ browser-compat: javascript.builtins.Intl.Segments
 
 An **`Intl.Segments`** instance is an iterable collection of the segments of a text string. It is returned by a call to the [`segment()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter/segment) method of an [`Intl.Segmenter`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter) object.
 
+{{EmbedInteractiveExample("pages/js/intl-segments-prototype-containing.html ")}}
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/intl/segments/index.md:16]:
```suggestion
{{EmbedInteractiveExample("pages/js/intl-segments-prototype-containing.html")}}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/reference/global_objects/intl/segmenter/index.md (hunk 1 above)
Comment (SphinxKnight) [reply files/en-us/web/javascript/reference/global_objects/intl/segmenter/index.md:16]: No reason at all, sorry for letting this slip. I'll apply the suggestions

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**